Projects/GLU/GLU01/GLU01-AWS100.md

```diff
@@ -0,0 +1,38 @@
+# GLU02-AWS100 - Introduction to SNS (Simple Notification Service)
+
+## Cloud Service Provider
+- Amazon Web Services
+
+## Difficulty
+- Level 100 (Introductory)
+
+## Project's Author(s)
+
+- [Edward Allen Mercado](https://twitter.com/edwardmercado_)
+
+## Objectives
+
+### You need to complete the following:
+
+- Create a SNS Topic
+- Subscribe to a SNS Topic
+- Receive email notification from the SNS Topic created
+
+### You need to answer the following:
+
+- What is SNS?
+- How Pub/Sub notifications work?
+- What are the different endpoints that SNS can send notification?
+
+## References
+
+- [AWS SNS Official Documentation](https://docs.aws.amazon.com/sns/latest/dg/welcome.html)
+
+## Costs
+
+- Included in the Free Tier
+- As part of AWS’s Free Usage Tier, new AWS customers will receive free 15 GB of data transfer out each month aggregated across all AWS services for one year.
```

Comment:
Egress data transfer does not count towards SNS, omit please. Just leave the "Included in the free tier" one

Reply:
actually, I got that one from sns pricing documentation. 

`** As part of AWS’s Free Usage Tier, new AWS customers will receive free 15 GB of data transfer out each month aggregated across all AWS services for one year.`

but no worries, I amended it now on my latest commit. Thank you!